core/tests/loadnext/src/executor.rs

```diff
@@ -85,11 +108,32 @@ impl Executor {
     async fn start_inner(&mut self) -> anyhow::Result<LoadtestResult> {
         tracing::info!("Initializing accounts");
         tracing::info!("Running for MASTER {:?}", self.pool.master_wallet.address());
+
+        if let Some(base_token) = self.base_token {
+            anyhow::ensure!(
+                base_token != self.config.main_token,
+                "Main testing token and chain's base token should be different"
+            );
+        }
+
+        tracing::info!(
+            "Running for chain based on {:?}",
+            self.base_token
+                .map(|token| token.to_string())
+                .unwrap_or("ETH".to_string())
+        );
+
         self.check_onchain_balance().await?;
-        self.mint().await?;
-        self.deposit_to_master().await?;
 
-        self.deposit_eth_to_paymaster().await?;
+        tracing::info!("Master Account: Minting Test ERC20 token...");
+        self.mint(self.config.main_token).await?;
```

Comment:
nit: it might be sometimes confusing to the reader on what is the difference between "main" and "base" tokens. I would suggest renaming the "main" token field

Reply:
It's used in workflows (e.g. [here](https://github.com/matter-labs/zksync-era-workflows/blob/5d959660d8a9c439b45a4f4d9c27af7993c6e035/.github/workflows/loadtest-reusable.yml#L123)) so probably easier to add a doc comment?